history/taskana-simplehistory-provider/src/test/java/acceptance/query/QueryHistoryAccTest.java

```diff
@@ -16,14 +16,14 @@
 import pro.taskana.spi.history.api.events.TaskanaHistoryEvent;
 
 /** Test for History queries. */
-public class QueryHistoryAccTest extends AbstractAccTest {
+class QueryHistoryAccTest extends AbstractAccTest {
 
   public QueryHistoryAccTest() {
     super();
   }
 
   @Test
-  public void should_ConfirmEquality_When_UsingListValuesAscendingAndDescending() {
+  void should_ConfirmEquality_When_UsingListValuesAscendingAndDescending() {
```

Comment:
true. Sonarcloud would otherwise raise an issue and ask us to remove the modifier. In order to reduce round trips for the future I simply removed them everywhere via replace in path